package.json

```diff
@@ -59,7 +60,8 @@
     "eslint:check": "eslint . --ext .js,.ts,.tsx --max-warnings=0",
     "prettier:check": "prettier --check .",
     "package:lib": "hygen package ts && npm run init",
-    "package:sanity-plugin": "hygen package sanity-plugin && npm run init"
+    "package:sanity-plugin": "hygen package sanity-plugin && npm run init",
+    "upgrade:sanity": "ncu /@sanity/ -u -t minor && lerna exec -- ncu /@sanity/ -u -t minor && npm install && npm run init"
```

Comment:
oi, hva gjør denne?

Reply:
Oppgraderer Sanity biblioteker i alle pakker ;) Har lagt til kort forklaring i README. Også - har brukt scripten for å oppdatere alt til 2.30.0 nå.

Reply:
Nice!